README-CONAN.md

```diff
@@ -117,25 +202,208 @@ os_build=Windows
 ```bash
-$ mkdir build
-$ cd build
+c:\....\exiv2\build> conan install .. --profile msvc2017Release64 --build missing
+c:\....\exiv2\build> cmake         .. -G "Visual Studio 15 2017 Win64" -DCMAKE_BUILD_TYPE=Release
+c:\....\exiv2\build> cmake --build .  --config Release
 ```
 
-Finally, we use the `conan install` command pointing to the directory where the `conanfile.py` is, to bring the
-dependencies:
+CMake provides 4 Generators.  The 64 and 32 bit Generators have different names:
+
+| Architecture | Visual Studio 2015 | Visual Studio 2017|
+|:---------|--------------------|--------------------|
+| 64 bit | "Visual Studio 14 2015 Win64"    | "Visual Studio 15 2017 Win64" |
+| 32 bit | "Visual Studio 14 2015"    | "Visual Studio 15 2017" |
+
+### Recommended settings for Visual Studio
+
+
+##### 64 bit Release Build
+
+| | Visual Studio 2015 | Visual Studio 2017|
+|:---------|--------------------|--------------------|
+| _**conan install .. --profile**_ | msvc2015Release64 | msvc2017Release64 |
+| _**cmake -G**_                   |  "Visual Studio 14 2015 Win64"    | "Visual Studio 15 2017 Win64" |
+| _**cmake**_                      | -DCMAKE\_BUILD\_TYPE=Release | -DCMAKE\_BUILD\_TYPE=Release |
+| _**profile**_ | arch=x86\_64  | arch=x86\_64 |
+| | arch\_build=x86\_64 | arch\_build=x86\_64 |
+| | build\_type=Release | build\_type=Release |
+| | compiler.runtime=MD | compiler.runtime=MD |
+| | compiler.version=14 | compiler.version=15 |
+| | compiler=Visual Studio | compiler=Visual Studio |
+| | os=Windows | os=Windows |
+| | os\_build=Windows | os\_build=Windows |
+
+##### Debug Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-------|-------|------|
+| _**conan install .. --profile**_ | msvc2015Debug64 | msvc2017Debug64 |
+| _**cmake**_ | -DCMAKE\_BUILD\_TYPE=Debug | -DCMAKE\_BUILD\_TYPE=Debug |
+| _**profile**_ | build_type=Debug | build_type=Debug |
+| | compiler.runtime=MDd | compiler.runtime=MDd |
+
+##### 32bit Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-----------|--------------------|--------------------|
+| _**conan install .. --profile**_ | msvc2015Release32 | msvc2017Release32 |
+| _**cmake -G**_ | "Visual Studio 14 2015" | "Visual Studio 15 2017" |
+| _**profile**_ | arch=x86 | arch=x86 |
+| | arch_build=x86 | arch_build=x86 |
+
+### Modifying build settings in the Visual Studio IDE
+
+**CAUTION** You can change the runtime library in Visual Studio once you open the
+solution generated by CMake.  If you have generated a project to use the dynamic run-time library
+with the profile setting `compiler.runtime=MD` (or MDd), Visual Studio allows you to change that setting.
+However the libraries (provided by conan) to which you link are tied
+to a specific configuration and you will obtain linking errors.
+
+If you want to work with the static runtime library `MT` (or `MTd`), set up the profiles and build in a different directory such as build-static.
+
+### Changing profile settings with the conan command
```

Comment:
I think this section can be confusing and it does not bring to much value to the document. I've been using conan for a long time, and rarely I need to use the `--settings` option. IMHO I would remove this section from the document.

Reply:
I agree.  I will remove this.

This was in the original version of this document.  I wondered if you'd put this into the document because this had caused you pain in the past.  One reason that Visual Studio Users dislike CMake is because it's often impossible to "tweak" settings in the Visual Studio IDE.  When they build, CMake is rerun, the SLN and VCPROJ files are rewritten and their changes are thrown away.